modal/cli/launch.py

```diff
@@ -55,6 +55,7 @@ def jupyter(
     timeout: int = 3600,
     image: str = "ubuntu:22.04",
     add_python: Optional[str] = "3.11",
+    no_python: Optional[bool] = False,
```

Comment:
It's typicall `--no-X` and `--X`, so `--no-add-python`.

Reply:
That's clever although I think the `--add` `--no-add` duality is specifically for _boolean_ typed flags right? Whereas `--add-python` has a mandatory argument?

Reply:
True, though I think it's still better than `--no-python` because there's still a duality here. `--add-python=""` looks weird.